### PR TITLE
Fix 'skip to end' link on DMCA

### PR DESCRIPTION
**Pull request by Katsute**
### Why:

Closes [issue link]

### What's being changed:

'Skip to end' link should refer to section g, not section f

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@Katsute Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/github/site-policy/dmca-takedown-policy.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ topics:
 
 Welcome to GitHub's Guide to the Digital Millennium Copyright Act, commonly known as the "DMCA." This page is not meant as a comprehensive primer to the statute. However, if you've received a DMCA takedown notice targeting content you've posted on GitHub or if you're a rights-holder looking to issue such a notice, this page will hopefully help to demystify the law a bit as well as our policies for complying with it.
 
-(If you just want to submit a notice, you can [skip to the end](#f-submitting-notices).)
+(If you just want to submit a notice, you can [skip to the end](#g-submitting-notices).)
```
Comment (hubwriter) [inline content/github/site-policy/dmca-takedown-policy.md:18]:
```suggestion
(If you just want to submit a notice, you can skip to "[G. Submitting notices](#g-submitting-notices).")
```

**Review by hubwriter — DISMISSED** [review]
@Katsute - Many thanks for submitting this correction. Your change was good but I've made a small change to use the subheading text.

Thanks again for helping to improve the GitHub documentation. Your assistance is very much appreciated. 👍

File: content/github/site-policy/dmca-takedown-policy.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ topics:
 
 Welcome to GitHub's Guide to the Digital Millennium Copyright Act, commonly known as the "DMCA." This page is not meant as a comprehensive primer to the statute. However, if you've received a DMCA takedown notice targeting content you've posted on GitHub or if you're a rights-holder looking to issue such a notice, this page will hopefully help to demystify the law a bit as well as our policies for complying with it.
 
-(If you just want to submit a notice, you can [skip to the end](#f-submitting-notices).)
+(If you just want to submit a notice, you can skip to "[G. Submitting notices](#g-submitting-notices).")
```
Comment (hubwriter) [inline content/github/site-policy/dmca-takedown-policy.md:18]:
```suggestion
(If you just want to submit a notice, you can skip to "[G. Submitting Notices](#g-submitting-notices).")
```

**Review by hubwriter — APPROVED** [review]

**Pull request merged**